src/ts/mapboxGeocoderControl.ts

```diff
@@ -41,7 +42,8 @@ module powerbi.extensibility.visual {
             this.geocoder = new MapboxGeocoder({
                 accessToken: this.accessToken,
                 zoom: this.zoom,
-                trackProximity: false // BUG in mapbox-gl-geocoder
+                trackProximity: true,
```

Comment:
Per docs this defaults to `true`, we could just remove? https://github.com/mapbox/mapbox-gl-geocoder/blob/master/API.md#parameters